src/input_cp2k_mp2.F

```diff
@@ -1190,9 +1190,18 @@ SUBROUTINE create_low_scaling(section)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
+      CALL keyword_create( &
+         keyword, __LOCATION__, name="EPS_FILTER", &
+         description="Determines a threshold for the DBCSR based multiply."// &
+         "Normally, this EPS_FILTER determines accuracy and timing of low-scaling RPA and GW calculations.", &
+         usage="EPS_FILTER 1.0E-10 ", type_of_var=real_t, &
+         default_r_val=1.0E-12_dp)
```

Comment:
I don't think there is a need to change this default. I think the previous default (`EPS_FILTER` from `&WFC_GPW`, `1.0E-09_dp`) is reasonably safe.

Reply:
Thanks for pointing this out, let's better keep it as you write.